src/NServiceBus.Core/Routing/ConfigureUniquelyAddressableQueue.cs

```diff
@@ -0,0 +1,21 @@
+﻿namespace NServiceBus
```

Comment:
yeah sorry, had v5 open at that time. As mentioned yesterday I don't have strong opinions on the file location. Whether it is in routing, pipeline or in the root doesn't really change anything and we can always move it in a nonbreaking way.